<ui version="4.0" >
 <class>materialParameters</class>
 <widget class="QWidget" name="materialParameters" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Material</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" colspan="2" >
         <widget class="PredefinedMaterials" name="predefinedMaterials" >
          <property name="toolTip" >
           <string>Choose a predefined material.</string>
          </property>
          <property name="editable" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QDoubleSpinBox" name="transparency" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="toolTip" >
           <string>Set the transparency of the surface (0.0=opaque, 1.0=transparent). If this is enabled, you should use the same materials for front and back side.</string>
          </property>
          <property name="maximum" >
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="ColorPicker" name="diffuse" >
          <property name="toolTip" >
           <string>Set the diffuse color.</string>
          </property>
          <property name="text" >
           <string> Diffuse</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="ColorPicker" name="emission" >
          <property name="toolTip" >
           <string>Set the emission/glow color. </string>
          </property>
          <property name="text" >
           <string> Emission</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="transparencyLabel" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Transparency:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_1" >
          <property name="text" >
           <string>Shininess:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="ColorPicker" name="specular" >
          <property name="toolTip" >
           <string>Set the color of specular highlights.</string>
          </property>
          <property name="text" >
           <string> Specular</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QDoubleSpinBox" name="shininess" >
          <property name="toolTip" >
           <string>Set the strength of the specular reflections.</string>
          </property>
          <property name="maximum" >
           <double>128.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="ColorPicker" name="ambient" >
          <property name="toolTip" >
           <string>Set the ambient color.</string>
          </property>
          <property name="text" >
           <string> Ambient</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QPushButton</extends>
   <header>ColorPicker.h</header>
  </customwidget>
  <customwidget>
   <class>PredefinedMaterials</class>
   <extends>QComboBox</extends>
   <header>PredefinedMaterials.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
